app/controllers/pages_controller.rb

```diff
@@ -2,5 +2,6 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: :home
 
   def home
+    @planets = Planet.first(4)
```

Comment:
only assign planets not owned by current user to @planets

Reply:
@fleveil done :)